sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -734,6 +734,11 @@ aditof::Status CameraItof::requestFrame(aditof::Frame *frame) {
         return status;
     }
 
+    if (m_dropFirstFrame) {
+        m_depthSensor->getFrame(frameDataLocation);
+        LOG(INFO) << "Dropped first frame";
+    }
```

Comment:
The overall implementation is good but there is a major problem here: this will make it so that every other frame will be dropped. For instance, if CameraItof::requestFrame() is called 4 times, m_depthSensor->getFrame() will be called 8 times, and the 1st, 3rd, ...7th frames will be dropped.

Reply:
Yes, you are right. I solved it by setting m_dropFirstFrame to false after first call of getFrame.